README.md

```diff
@@ -1,348 +1,143 @@
 # Mortar
 
-Mortar eases the use of [Dagger][dagger] to divide Android apps into
-composable modules. It is not a framework: there are no abstract Application
-or Activity classes here. Mortar eschews magic. Rather, Mortar is a simple
-library that makes it easy to pair thin views with dedicated controllers
-(Presenters), isolated from most of the vagaries of the Activity life cycle.
-The patterns it encourages have evolved in parallel across several Android
-teams at Square.
-
-Mortar relies on Dagger, and of course the Android runtime, but those are its
-only direct dependencies. That said, it works very well with
-[Retrofit][retrofit], [Flow][flow] and [RxJava][rxjava].
-[Butterknife][butterknife] can be a fun partner too. (Use of all of these
-libraries is illustrated in the sample app.)
+## What's a Mortar?
+
+Mortar provides a simplified, composable overlay for the Android lifecycle,
+to aid in the use of [Views as the modular unit of Android applications][rant].
+It turns [Context#getSystemService][services] into an a la carte supplier
+of services like dependency injection, bundle persistence, and whatever else
+your app needs to provide itself.
+
+One of the most useful services Mortar can provide is its 
+[BundleService][bundle-service], which gives  any View class safe
+access to the Activity lifecycle's persistence bundle.
+For fans of the [Model View Presenter][mvp] pattern, this service includes a [Presenter][presenter]
+class that builds on that service. These Presenters are completely isolated
+from View concerns, but can still be persisted by the BundleService. They're
+particularly good at surviving configuration changes, weathering the storm 
+as Android destroys your portrait Activity and Views and replaces them 
+with landscape doppelgangers. 
+
+Mortar can similarly make [Dagger][dagger] ObjectGraphs visible as system
+services, or [Dagger2][dagger2] Components, or any other service that might be
```

Comment:
Dropping the any other service for now. I'll be making more passes over this later.